packages/contentful/src/crn/queries/users.queries.ts

```diff
@@ -153,10 +153,44 @@ export const usersContentQueryFragment = gql`
           }
         }
       }
+      researchOutputsCollection {
```

Comment:
Maybe we can make this piece composable so we don't make the query more complex, since this information is not used by the hub, just the public api.